### ISSUE TITLE
Shadow gone away when adding Slidable Widget

### ISSUE THREAD

**Issue by Tomic-Riedel** [letsar/flutter_slidable#279]
Hello guys!
I have a little problem with the Slidable Package.

I have a container with a boxshadow which looks like this:
<img width="247" alt="Screenshot 2021-12-04 at 17 17 34" src="https://user-images.githubusercontent.com/64801984/144716673-ee2465b4-971b-4e59-93c4-3f7ab9005ab9.png">

Unfortunately, as soon as I add the Slidable widget, it looks completely different. For some reason the boxshadow is "compressed", if you can say that. Here is a picture of how it looks with the Slidable Widget:
<img width="236" alt="Screenshot 2021-12-04 at 17 31 15" src="https://user-images.githubusercontent.com/64801984/144717065-7c0539df-a9d1-455e-ba56-71928ff52356.png">

Here is my current code:

```dart
Slidable(
        closeOnScroll: true,
        endActionPane: ActionPane(
          motion: DrawerMotion(),
          extentRatio: 0.3,
          children: [
            SlidableAction(
              onPressed: (context) {},
              backgroundColor: Colors.transparent,
              foregroundColor: Colors.red,
              icon: PhosphorIcons.trash,
              autoClose: true,
            ),
          ],
        ),
        child: Container(
          padding: const EdgeInsets.all(20),
          decoration: BoxDecoration(
            borderRadius: BorderRadius.circular(20),
            color: Colors.white,
            boxShadow: [
              BoxShadow(
                offset: const Offset(8, 10),
                blurRadius: 10,
                color: kShadowColor,
              ),
            ],
          ),
          child: anyChild
       ),
),
```

Thanks for your help in advance!

**Comment by Nothehi** [comment]
did you solve this problem? I have the same problem!

**Comment by letsar** [comment]
Thank you for raising the issue. I fixed it, and it will be available in a next release.

**Comment by letsar** [comment]
Fixed in version 1.3.1.
